### DISCUSSION TITLE
FireFox PWA Introduces Malware + Error Occurs #479

### DISCUSSION THREAD

**Discussion by nanashinasubi** [filips123/PWAsForFirefox#481] [Q&A] [answered]
I really wanted this application to work. However, every time I download the installation packages and run the system, Microsoft virus & threat protection identifies severe Trojan malware in the files. When the system removes the Trojan malware, the extension has an error and I cannot save any websites as PWAs. Any workarounds? 
![image](https://github.com/filips123/PWAsForFirefox/assets/164282731/09531ca0-b366-436b-9f3c-8729eeb21659)
![image](https://github.com/filips123/PWAsForFirefox/assets/164282731/366da256-27f7-467f-aad0-1e38e439844b)

**Comment by filips123** [comment, marked as answer]
It is a false positive. You need to update your malware definitions (and probably reinstall PWAsForFirefox). See more details in #480.
